translib/utils/utils.go

```diff
@@ -21,7 +21,6 @@ package utils
 
 import (
 	"fmt"
-	//"github.com/Azure/sonic-mgmt-common/translib/db"
```

Comment:
why we need this change? this import remove is already present in community file. Check for white-space